au/math_test.cc

```diff
@@ -713,8 +713,8 @@ TEST(CeilIn, SupportsDifferentOutputTypes) {
 }
 
 TEST(InverseAs, HandlesIntegerRepCorrectly) {
-    constexpr auto period = inverse_as(milli(seconds), hertz(40));
```

Comment:
Comment: I guess we are using the integer division operator so there may be data loss for cases like `hertz(41)`. With `hertz(40)`, 1/40 is a multiple of 0.005, but that can only be determined at compile time.

Reply:
`hertz(41)` is technically still lossless in the sense that the round trip is guaranteed to recover the initial value for everything up to 1,000.  (It's true that the integer result itself won't be exact, of course.)